flang/lib/Lower/IO.cpp

```diff
@@ -562,37 +564,9 @@ mlir::Value genIOOption<Fortran::parser::IoControlSpec::Rec>(
 }
 
 //===----------------------------------------------------------------------===//
-// Gathering of I/O statement error specifiers (if any).
+// Gather I/O statement condition specifier information (if any).
```

Comment:
I don't really know what "condition specifier" means. Is that from the standard?

Reply:
Yes, this is from clause 12.11, which has the title _Error, end-of-record, and end-of-file conditions_.  "end-of-record" and "end-of-file" are not in general errors.

Reply:
Thanks.

The original name wasn't so great. I agree.